.github/workflows/run-pytest.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         config: [ {python: '3.8', dependencies: 'newest'},
                   {python: '3.9', dependencies: 'newest'},
                   {python: '3.10', dependencies: 'newest'},
+                  {python: '3.11', dependencies: 'newest'},
                   {python: '3.11', dependencies: 'minimal'},
```

Comment:
Now that you have the ability to test with the newest dependencies on python 3.11, do you still want to test python 3.11 with the minimal dependencies? Is having a "minimal" classification even necessary anymore? If not, much of the logic in this file can likely be simplified.

Reply:
We keep one test config running on 'minimal' so that we know the test suite will pass without dependencies that are listed as optional. That ensures that 1) we properly skip tests that require optional dependencies to function, and 2) no hard dependency is accidentally listed as optional. In practice, since signac is pure Python it would probably be safe to just run one of the 4 Python versions as minimal rather than running both minimal and newest for 3.11, I don't have a strong opinion there.

Reply:
We have historically run the latest Python version with both “newest” and “minimal” dependencies. I think we should stick to that. Minimal tests run super fast.